This run started at: 
1579043740
The zcashd version is: Zcash Daemon version v2.1.1-rc1-02706325c

In order to ensure you are adequately protecting your privacy when using Zcash,
please see <https://z.cash/support/security/>.

Copyright (C) 2009-2020 The Bitcoin Core Developers
Copyright (C) 2015-2020 The Zcash Developers

This is experimental software.

Distributed under the MIT software license, see the accompanying file COPYING
or <https://www.opensource.org/licenses/mit-license.php>.

This product includes software developed by the OpenSSL Project for use in the
OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written
by Eric Young.
=== Running testscript intermittent_failures.py ===
  File "/home/pherder/zcashes/intermittenttestfailures/master/qa/rpc-tests/test_framework/test_framework.py", line 121, in main
    self.run_test()
  File "/home/pherder/zcashes/intermittenttestfailures/master/qa/rpc-tests/intermittent_failures.py", line 43, in run_test
    txid_2 = wait_and_assert_operationid_status(self.nodes[0], myopid)
  File "/home/pherder/zcashes/intermittenttestfailures/master/qa/rpc-tests/test_framework/util.py", line 446, in wait_and_assert_operationid_status
    result = wait_and_assert_operationid_status_result(node, myopid, in_status, in_errormsg, timeout)
  File "/home/pherder/zcashes/intermittenttestfailures/master/qa/rpc-tests/test_framework/util.py", line 437, in wait_and_assert_operationid_status_result
    assert_equal(in_errormsg, errormsg)
  File "/home/pherder/zcashes/intermittenttestfailures/master/qa/rpc-tests/test_framework/util.py", line 387, in assert_equal
    raise AssertionError("(left == right)%s\n  left: <%s>\n right: <%s>" % (message, str(expected), str(actual)))
Initializing test directory /tmp/test2iVsEd
waiting for async operation opid-cbe1677d-43a0-4dea-ac25-b4b1f8f392c4
waiting for async operation opid-f39d565b-87b7-4d36-aaba-9d6747d0e4e5
waiting for async operation opid-661da2a6-3426-4158-b1e4-19978d5f3f26
Assertion failed: (left == right)
  left: <None>
 right: <Insufficient funds, no unspent notes found for zaddr from address.>
Stopping nodes
Cleaning up
Failed
!!! FAIL: intermittent_failures.py (79s) !!!



Tests completed: 1
successes 0; failures: 1

Failing tests: intermittent_failures.py
